VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/command-centre/command-centre.component.ts

```diff
@@ -377,6 +383,12 @@ export class CommandCentreComponent implements OnInit, OnDestroy {
         }
     }
 
+    handleHearingDetailUpdate(hearingDetailMessage: HearingDetailsUpdatedMessage) {
+        if (hearingDetailMessage.conference) {
```

Comment:
Rather than making a fresh API call, it would make more sense to find the conference by id in the list and update the object

Reply:
ok I will try